Notebooks/Russian Twitter Corpus/.ipynb_checkpoints/rusain-corpus-tweets-analysis-checkpoint.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
Remove `.ipynb_checkpoints` and `__pycache__` from our repository but not from our project. This kind of folders looks like a configuration for your system :)